docs/csharp/fundamentals/exceptions/index.md

```diff
@@ -30,7 +30,7 @@ Exceptions have the following properties:
 - Exceptions can be explicitly generated by a program by using the `throw` keyword.
 - Exception objects contain detailed information about the error, such as the state of the call stack and a text description of the error.
 - Code in a `finally` block is executed regardless of if an exception is thrown. Use a `finally` block to release resources, for example to close any streams or files that were opened in the `try` block.
-- Managed exceptions in .NET are implemented on top of the Win32 structured exception handling mechanism. For more information, see [Structured Exception Handling (C/C++)](/cpp/cpp/structured-exception-handling-c-cpp) and [A Crash Course on the Depths of Win32 Structured Exception Handling](http://bytepointer.com/resources/pietrek_crash_course_depths_of_win32_seh.htm).
+- Managed exceptions in .NET are implemented on top of the Win32 structured exception handling mechanism. For more information, see [Structured Exception Handling (C/C++)](/cpp/cpp/structured-exception-handling-c-cpp).
```

Comment:
I think we should point to the archived version of this article. It's a better formatted copy of the Microsoft Systems Journal article (available directly from [archive.org](https://web.archive.org/web/20180115191634/http://www.microsoft.com:80/msj/0197/exception/exception.aspx)). From my quick research, it seems this is the definitive article about this stuff, even if it's from 1997!!

```suggestion
- Managed exceptions in .NET are implemented on top of the Win32 structured exception handling mechanism. For more information, see [Structured Exception Handling (C/C++)](/cpp/cpp/structured-exception-handling-c-cpp) and [A Crash Course on the Depths of Win32 Structured Exception Handling](https://web.archive.org/web/20240204063742/http://bytepointer.com/resources/pietrek_crash_course_depths_of_win32_seh.htm).
```